notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -428,13 +425,14 @@
    },
    "outputs": [],
    "source": [
-    "num_classes = 200  # 200 is for Tiny ImageNet, default is 1000 for imagenet\n",
+    "num_classes = 200  # 200 is for Tiny ImageNet, default is 1000 for ImageNet\n",
     "init_lr = 1e-4\n",
     "batch_size = 128\n",
     "epochs = 4\n",
     "\n",
-    "# create model\n",
-    "model = models.resnet18(pretrained=True)\n",
+    "# To train the Tiny ImageNet dataset yourself on a model that is pretrained on ImageNet,\n",
+    "# set pretrained to False in the Imports and Settings section, and to True in the line below.\n",
+    "model = models.resnet18(pretrained=False)\n",
```

Comment:
It would be great to have a single variable that is responsible for downloading a pretrained weights whether from TorchVision or from openvino.org

But to avoid a confusing line: `model = models.resnet18(pretrained=not pretrained)`
I'd rename `pretrained` in Imports and Settings section to `pretrained_on_tiny_imagenet`
```suggestion
    "model = models.resnet18(pretrained=not pretrained_on_tiny_imagenet)\n",
```

Reply:
Thanks that is much better! I changed it (in a new commit because I also had to change it at the top).